README.md

```diff
@@ -1,2 +1,108 @@
+web application for a company that provides commercial and scientific space travel services. The application will allow users to book rockets and join selected space missions.
+
+## Built With
+
+- HTML, CSS, JS
+- React, JSX, Redux
+- Gitflow
+
+
+## Live Demo
+
+
+
+### Prerequisites
+- IDE(code editor) like: **Vscode**, **Sublime**, etc. 
+- [Git](https://www.linode.com/docs/guides/how-to-install-git-on-linux-mac-and-windows/)
+
+### Setup
+1. Download the **Zip** file or clone the repo with:
+```bash
+git clone https://github.com/yetemegn-telaye/Space-travelers-hub.git
+```
+2. To access cloned directory run:
+```bash
+cd Space-travelers-hub
+
+
+
+### Install
+> To install linters, react and other project's dependencies run:
+```bash
+npm install
+```
+## Run
+```bash
+1. To run this project :
+npm start
+2. open the urs localhost:8080 on your browser
+
+```
+
+### `npm run build`
+
+Builds the app for production to the `build` folder.\
+
+
+### `npm run eject`
+
+**Note: this is a one-way operation. Once you `eject`, you can't go back!**
+
+If you aren't satisfied with the build tool and configuration choices, you can `eject` at any time. This command will remove the single build dependency from your project.
+
+Instead, it will copy all the configuration files and the transitive dependencies (webpack, Babel, ESLint, etc) right into your project so you have full control over them. All of the commands except `eject` will still work, but they will point to the copied scripts so you can tweak them. At this point you're on your own.
+
+You don't have to ever use `eject`. The curated feature set is suitable for small and middle deployments, and you shouldn't feel obligated to use this feature. However we understand that this tool wouldn't be useful if you couldn't customize it when you are ready for it.
+
+
+### `npm test`
+
+Launches the test runner in the interactive watch mode for React.\
+
+## Run Test for Linters
+
+For tracking linter errors locally you need to follow these steps:
+
+1. For tracking linter errors in HTML files run:
+```bash 
+npx hint .
+```
+
+2. For tracking linter errors in CSS or SASS files run:
+
+```bash
+npx stylelint "\*_/_.{css,scss}"
+```
+
+3. For tracking linter errors in JavaScript files run:
+
+```bash
+npx eslint .
+```
+
+## Authors
+
+👤 **Yetemegn Lemma**
+
+[GitHub](https://github.com/yetemegn-telaye) | [Twitter](https://twitter.com/Yetemegn19) | [LinkedIn](linkedin.com/in/yetemegn-lemma)
+
+👤 **Chukwuma Paul Mosanya**
+
+[GitHub](https://github.com/blase147) | [Twitter](https://twitter.com/ChukwumaMosanya) | [LinkedIn](https://www.linkedin.com/in/chukwuma-mosanya)
+
+👤 **Ayodeji Abidemi**
+
+[GitHub](https://github.com/demix007) | [Twitter](https://twitter.com/Dat_dope_demix) | [LinkedIn](http://linkedin.com/in/ayodejiabidemi)
+
+
+## Show your support
+
+Give a ⭐ if you like this project!
+
+
+## 📝 License
+
+This project is [MIT](./MIT.md) licensed.
```

Comment:
- [ ] Kindly add the MIT file, this link is broken.
- [ ] Kindly add some of the missing sections of the README file [template](https://github.com/microverseinc/readme-template) and fill them out with information about your project 👍